### DISCUSSION TITLE
Disable commenting via an email reply?

### DISCUSSION THREAD

**Discussion by bseib** [community/community#35715] [Projects and Issues]
Replying to a github issue comment via email can appear messy and difficult to read unless the senders consistently take care with their formatting. This ultimately costs time and money with regard to any of:
  - reading/understanding ill-formatted email-reply comments
  - reformatting those comments
  - meta communication reminding people to please format stuff correctly

As an organization we would like to simply enforce a policy where all comments are made via the website, which will eliminate the above costs. The email notifications already contain a link that navigates to the issue just fine. We just need a way to disable the respond-by-email-commenting feature on a per repository basis. Essentially just send those notifications from `no-reply@github.com` like once upon a time: https://github.blog/2011-03-10-reply-to-comments-from-email/

Could we add this as a setting?

_(I could not find this as an existing feature, nor a discussion about it. Hope I've not overlooked anything.)_

**Comment by llvm-beanz** [comment]
When combined with Outlook's new "email reactions" this creates a bad experience in the GitHub UI. Please allow us to disable this feature.

**Comment by vkbo** [comment]
The email reply is still broken after years of issues. I don't care if it is fixed. I would like to just turn it off. Due to the numerous email clients out there, you get all sorts of additional junk in the replies, it does not handle pasted images from screenshots, so the messages are sometimes not understandable, and when replying to subthreads in discussions, you generally get a new thread instead (I'm not sure if it always happens, but it certainly *does* happen) which breaks the entire threading of a discussion.

**Comment by bseib** [comment]
@emmaviolet, @acadavid, @geramirez -- are any of you involved in feature work similar to this request? (I did some guess work to pick you guys to at-mention -- sorry if this area is not what you work on.) I suspect that most of the effort would be to add the controls for this in the repo settings. Actually sending comment notifications from `no-reply` would be the easy part.

Any chance you could help promote/triage this request? Or point us to someone who can?

**Reply by geramirez** [reply]
@kellyarwine is the new PM on notifications

**Reply by bseib** [reply]
Hi @kellyarwine -- how can we get this on your radar? As issues go, this one is not terribly complicated, and I think it would create real savings for people.
